src/index.d.ts

```diff
@@ -1,53 +1,169 @@
 // Copyright (c) Jupyter Development Team.
```

Comment:
Ahh, sorry, I forgot that input/output area have existing code. Hmmm,
@jdfreder - thoughts on how to manage this? While I think the existing code
was a good experiment, I don't think it is rich enough to survive as is -
at the same time, we don't have a functional replacement yet. Hmmm.

On Tue, Nov 3, 2015 at 8:20 PM, Steven Silvester <notifications@github.com>
wrote:

> In src/index.d.ts
> <https://github.com/jupyter/jupyter-js-output-area/pull/18#discussion_r43831701>
> :
>
> > @@ -1,53 +1,169 @@
> >  // Copyright (c) Jupyter Development Team.
>
> Because the rest of the repo is JS. [image:
> :stuck_out_tongue_winking_eye:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyter-js-output-area/pull/18/files#r43831701>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com